luci/luci-app-ffwizard-falter/po/en/ffwizard-falter.po

```diff
@@ -93,19 +93,19 @@ msgid ""
 "muessen. <br/>Diese Einstellung bezieht sich auf den WLAN-AP, <em>nicht</em> "
 "das Adhoc-Netzwerk, das fuer das Freifunk-Meshing benutzt wird."
 msgstr ""
-"You should leave the SSID at <em>berlin.freifunk.net</em>, if there is an "
-"access point with the same SSID nearby. Thus, people can connect to the "
-"Freifunk network in the whole area, without having to connect to every node "
-"individually. <br/>This option influences the WiFi-AP, <em>not</em> the mesh-"
-"network for the mesh connections."
+"You should leave the SSID at <em>berlin.freifunk.net</em>, as long as there "
+"is not an access point with the same SSID nearby. Thus, people can connect "
+"to the Freifunk network in the whole area, without having to connect to "
+"every node individually. <br/>This option influences the WiFi-AP, <em>not</"
+"em> the mesh-network for the mesh connections."
```

Comment:
maybe don't break the </em> over two lines.

Reply:
I didn't break that by intention. That break probably that break was introduces by the translation editor itself. I assume that it shouldn't matter, but otherwise it shouldn't break anything, if I correct this.

Reply:
corrected in the new commit.